packages/ozone/src/db/migrations/20240228T003647759Z-add-label-sigs.ts

```diff
@@ -0,0 +1,17 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('label')
+    .addColumn('sig', sql`bytea`)
+    .execute()
+  await db.schema
+    .alterTable('label')
+    .addColumn('signingKey', 'varchar')
```

Comment:
It's not a big issue, but if it's easy to avoid storing the full signing key with each label that would be pretty nice.

One way it could work: during startup the public signing key could be upserted into a table and exchanged for an id.  Then the labels table stores the id.  You never really need to join over to that table, I don't think, just used once at startup.  Definitely not necessary, but figured I'd toss it out there!

Reply:
yeah that's a good point - this is unnecessarily bulky

i'll take a stab at it 👌